apitest/src/main/resources/injicertify/SunBirdCNegative/AuthorizationCodeSunBirdCNeg/AuthorizationCodeSunBirdCNeg.yml

```diff
@@ -0,0 +1,17 @@
+AuthorizationCodeSunBirdC:
+   InjiCertify_ESignet_AuthorizationCode_SunBirdC_All_Valid_Smoke_sid:
```

Comment:
is this testcase name is unique?

Reply:
Description and unique identifier missing

Reply:
Updated the changes